src/dev/precommit_hook/casing_check_config.js

```diff
@@ -98,6 +98,7 @@ export const IGNORE_DIRECTORY_GLOBS = [
   ...KEBAB_CASE_DIRECTORY_GLOBS,
   'src/babel-*',
   'packages/*',
+  'packages/core/server/*',
```

Comment:
@elastic/kibana-operations Given we have a nested structure and that the actual package folders will be under `packages/core/server|public|common/`, I thought it was ok to add this folder to the list of ignored folder for snake_casing, as I think it's better to have `packages/core/server/my-core-package` rather than `packages/core/server/my_core_package`, for consistency with the naming of packages being directly under `/packages`.

Do that make sense to you?

Reply:
It makes sense to me. @spalger maybe this is something we want to address on our RFC what do you think?

Reply:
Would be great ihmo